tests/features/joinup_search/group_search.feature

```diff
@@ -0,0 +1,144 @@
+@api @terms
+Feature: Search inside groups
+  In order to quickly find content inside the group I am currently perusing
```

Comment:
"perusing" :joy: 
Now the devs will also require a proficiency in English to be able to read through the comments :joy:

Reply:
We shall require our developers to study some additional materials henceforth.

https://www.youtube.com/watch?v=Nj0Rh__1kDw

Reply:
If the Queen does not understand the behat scenario, then the behat scenario is rejected :joy: